src/components/frame/v5/pages/MembersPage/partials/SubNavigation/SubNavigation.module.css

```diff
@@ -0,0 +1,3 @@
+.navigationButton {
+  @apply w-full flex items-center py-2 px-4 hover:bg-gray-50 hover:font-medium transition-all duration-normal rounded;
```

Comment:
Are you sure we should use font medium on hover here? It causes a shift of hovered elements.

Reply:
It's added on designs, so I will leave it for now